### PR TITLE
Feature bootstrap-init

### PR DESCRIPTION
**Pull request by wantguns**
Bootstrap System-As-Root Android's `init` inside a LXC container.

### CODE REVIEW

**Comment by heroxbd** [comment]
Any progress in this PR?

File: sys-kernel/linux-headers/Manifest [hunk 1]
```diff
@@ -1,2 +1,2 @@
-EBUILD linux-headers-3.10.73.ebuild 1161 BLAKE2B b44eac62853b663f77744efeecbf624c15fa94e71a7e94805174426e9e6fd224100ce784ae5866ef9db9969471c397bb685531c0cc530e87e085218697979b1d SHA512 c2bd268881a30d59741aad0be1d422b341b7ca3d9cc811b7f2d22befb90b13b114b376901fc6f37984176de0bd666afb36cf565ed98cec32c78c713d5a298e19
```
Comment (KireinaHoro) [inline sys-kernel/linux-headers/Manifest:1]: I don't think we should replace the 3.10 headers for angler with that for the new device.  Consider adding a new ebuild with better versioning, or splitting the header package altogether.

File: sys-kernel/linux-headers/linux-headers-4.4.226.ebuild [hunk 2]
```diff
@@ -10,14 +10,14 @@ detect_version
 detect_arch
 
 inherit git-r3
-EGIT_REPO_URI="https://github.com/KireinaHoro/android_kernel_huawei_angler.git"
```
Comment (KireinaHoro) [inline sys-kernel/linux-headers/linux-headers-4.4.226.ebuild:13]: Same as the comments on the Manifest file.

**Review by KireinaHoro — CHANGES_REQUESTED** [review]

File: sys-kernel/linux-headers/Manifest (hunk 1 above)
Comment (wantguns) [reply sys-kernel/linux-headers/Manifest:1]: I don't remember why I didn't do this earlier. Sure, will do that.

**Comment by wantguns** [comment]
> Any progress in this PR?

I had my priorities set in some other projects right now. I will get this PR ready ASAP. Sorry for the inconvenience.

**Comment by wantguns** [comment]
As discussed in the IRC channel, I am sending the patches via mail to ease the entire process. This branch's version history is too ugly.

**Pull request closed without merging**